pages/index.vue

```diff
@@ -86,13 +102,42 @@ const scales = {
   locrian
 };
 
+const major =           [4, 3];
+const major7 =          [4, 3, 4];
+const major9 =          [4, 3, 4, 3];
+const major11 =         [4, 3, 4, 3, 3];
+const dominant7 =       [4, 3, 3];
```

Comment:
This is missing the 9th, 11th, and 13th versions of the dominant chord.
Makes me think of building a "Chord builder". That way we can add crazy stuffz like altered versions of the chord. Selecting if we want to shapen or flatten any note of the chord would be cool.